### DISCUSSION TITLE
New users not able to login after image upgrade

### DISCUSSION THREAD

**Discussion by todor-ops** [outline/outline#2681] [Self Hosting] [answered]
Docker image was upgraded to later version after I accidentally deleted slack app for auth. After docker image upgrade, login started working again, but new users are not able to login for first time. Displayed error is: "Authentication failed - we were unable to sign you in at this time. Please try later".      Tried with new G-Suite login, with same result.

**Comment by todor-ops** [comment]
yarn sequelize:migrate commands did not help with environment variables passed with commands, tried several from issues and discussions.

**Comment by tommoor** [comment, marked as answer]
> Docker image was upgraded to later version after I accidentally deleted slack app for auth

There's a lot going on here, how many versions did you jump?

> Authentication failed - we were unable to sign you in at this time

The underlying error should be visible in your server logs

**Reply by todor-ops** [reply]
Initial installation was created using docker compose in january 2020, probably v0.39. It was working fine until january 2021. when I accidentally deleted slack app for auth, and new slack ID wasn't working. I updated image to latest at that moment, v0.53. Later we noticed new users can not manage to login for the first time with the error from the description. Existing users were ok. We are now on the latest image and .env file version. Can you please point me where to find logs?

**Reply by todor-ops** [reply]
This is resolved, using latest image version, migrate commands and deleting manually affected users from database.
